packages/ng/date/humanize/humanize.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform, inject } from '@angular/core';
+import { Observable, map } from 'rxjs';
+import { LuHumanizeFormatter } from './humanize.formatter';
+import { LuRelativeTimeFormatUnit } from './humanize.model';
+import { relativeTimeTimer } from './humanize.utils';
+
+@Pipe({
+	name: 'luHumanize',
+	standalone: true,
+})
+export class LuHumanizePipe implements PipeTransform {
```

Comment:
Should `date` be included in the name? Many things can be humanized :D 

`luHumanizeDate` / `luDateHumanizer` / `luHumanizedDate` (😬)